src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -457,6 +466,13 @@ def engine_forward(
                     ):
                         break
 
+                    if callback is not None and not callback(token):
+                        _LOGGER.info(
```

Comment:
This should be a debug log. Shouldn't log this on each inference

Reply:
No, this *only* logs when the callback returns False; which should be done right?